app/src/main/kotlin/cz/covid19cz/erouska/ui/main/MainActivity.kt

```diff
@@ -85,7 +92,10 @@ class MainActivity :
             }
         })
 
-        viewModel.serviceRunning.value = CovidService.isRunning(this)
+        val isRunning = CovidService.isRunning(this)
+
+        viewModel.serviceRunning.value = isRunning
+        shortcutsManager.updateShortcuts(isRunning)
```

Comment:
Do you also need to update shortcuts from here when the service does it?

Reply:
I do. I noticed that when the scanning is paused and the app starts, the service is not started therefore `updateShortcuts` is not called.

Reply:
But the shortcut should be in the correct state already, shouldn't it?

Reply:
Well, in theory...